lib/caller.js

```diff
@@ -21,4 +21,6 @@ module.exports = function getCaller () {
       return file
     }
   }
+
+  return stack[2]?.getFileName()
```

Comment:
Don't use the `?` as it's not in node v12.

Reply:
Also, a comment explaining what this is would be very helpful.